oidc-ui/src/index.js

```diff
@@ -7,9 +7,9 @@ import "./i18n";
 import 'react-tooltip/dist/react-tooltip.css'
 
 ReactDOM.createRoot(document.getElementById("root")).render(
-  <React.StrictMode>
+  // <React.StrictMode>
```

Comment:
These changes should not be committed

Reply:
resolve in this [eb6b155](https://github.com/mosip/esignet/pull/367/commits/eb6b1555f18a59517440f34b91e15470a8e3273a)